src/lib/blockheights.js

```diff
@@ -31,10 +35,34 @@ const compareBlockHeightsCron = async () => {
   let matchingBlockHeights = 0;
 
   if (localBlockheight === "0")
+  {
```

Comment:
This is sms logic, which probably shouldn't be mixed with the blockHeight logic. Also this code is repeated ~4 times.

My suggestion is to lift this code into this structure:

```
lib/
- sms/
-- twilio.js (as you already have done)
-- textMagic.js (as you already have done)
-- index.js
```

index.js would then export a function `sendSMS()` in which this below logic gets moved into.

e.g.

```
// import your local sendTwilio
// import your local sendTextMagic

export const sendSMS = (message: string) => {
  // check last sent
  //  if twilio is enabled -> sendTwilio(message)
  // if sendTextMagic is enabled -> sendTextMagic(message)
}:
```

Then in this file, whenever you have to send an sms, you just call the function sendSMS(message) and it will take care to send the sms or not and with which provider. In this way you will be repeating code and the code becomes more readable :)